File: 3541-find-most-frequent-vowel-and-consonant/3541-find-most-frequent-vowel-and-consonant.java
class Solution {
    public int maxFreqSum(String s) {

        int n = s.length();
        int vm = 0;
        int cm=0;

        int [] freq = new int[256];
        for(int i=0; i<n; i++)
        {
            char ch = s.charAt(i);
            freq[ch]++;
           
        }
        for(int i=0; i<256; i++)
        {
            if(freq[i]>0)
            {
                char ch = (char) i;
                if(ch=='a'||ch=='e'||ch=='i'||ch=='o'||ch=='u')
                {
                vm = Math.max(freq[i],vm);
                }
                else
                {
                    cm = Math.max(freq[i],cm);

                }
            }
        }
        return vm+cm;
        
    }
}

File: 3541-find-most-frequent-vowel-and-consonant/3541-find-most-frequent-vowel-and-consonant.py
class Solution:
    def maxFreqSum(self, s: str) -> int:

        f = {}
        ch = list(s)
        for x in ch:
            if x in f:
                f[x] += 1
            else:
                f[x] = 1
        
        mv = 0
        mc =0
        for c in f:
            if c in "aeiou":
                if f[c] > mv:
                    mv = f[c]
            else:
                if f[c]> mc:
                    mc = f[c]
        return mv+mc
            


        
        
        